_drafts/A BLE Primer.md

```diff
@@ -0,0 +1,1202 @@
+---
+title: "A BLE Primer"
+author: mafaneh
+---
+
+## Introduction (.. and a personal story)
+I used to be obsessed with audio quality and finding the best sounding headphones and earphones. I was a Westone customer for many years (a US-based company known for their high quality products in different applications including: hearing aids, music, hearing protection, and military). In fact, if you're looking for great quality sounding earphones, I would highly recommend you check them out (no affiliation, just a happy customer!): www.westone.com.
+
+When Apple released the AirPods in December of 2016, I jumped on the bandwagon and pre-ordered them right off the bat. Apple sold me on the simplicity of connectivity, the relatively long battery life, and reliability.
+
+Fast forward 2.5 years and I have no regrets! Sure, there were tradeoffs and I miss the high fidelity sound from my Westone earphones (which I still carry in my backpack), but for me the benefits outweighed the drawbacks. I sacrificed quality in favor of convenience and practicality. I suspect there are others who would do the same and still be happy!
+
+So you may be asking, where am I going with all this? 
+
+In my opinion, this is what will help wireless succeed in the marketplace (especially for consumer products). For industrial, enterprise, and commercial, it will have to prove its robustness, reliability, and more importantly its ROI.
+
+People are willing to sacrifice some quality and other aspects (they wouldn't consider sacrificing) when they try wireless. The convenience factor is huge! Unfortunately, it sometimes comes at a steep price (like in the AirPods example). Over time, however, wireless will only get cheaper and more affordable.
+
+Wireless is changing the world, and it's providing many benefits that help make our lives easier. I'm not talking just about the consumer applications, but rather applications that make your life simpler and easier and completely in the background. Technology is best utilized when it happens in the background without the user ever being aware of its existence.
+
+Wireless provides the following benefits:
+
+- Removing wires.
+- Freedom and flexibility.
+- New use cases. Objects that cannot be connected with a wire, due to design factors or physical size limitations, can still be connected to other devices via wireless technologies.
+- Convenience, especially with remote or long-distance access.
+- It has the potential of saving lives! Think of a use case where an array of sensors are placed on the top of an electric pole, on the edge of a cliff, or even in a dam. Having the ability to collect data and control devices placed in these unsafe areas can reduce the safety risk significantly. 
+
+Of course nothing is perfect, so wireless has its drawbacks too:
+
+- Sacrificing quality in some cases (due to compression and limited bandwidth).
```

Comment:
I'm not completely sold on this introduction. IMO it risks losing technical folks who are here for the BLE part. Starting with the "Wireless Technologies and the Internet of Things" section would be my personal choice (@tyhoff let me know what you think)

Reply:
Yes. I had a feeling it was a bit off-topic and too much for this post.